config/environments/production.rb

```diff
@@ -154,7 +154,8 @@
   ###################################################################
   # Location of the public and private key for the git user on the system
   GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
-                        private_key: "/home/git/.ssh/id_rsa" }
+                        private_key: "/home/git/.ssh/id_rsa",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.